app/comments/get.controller.test.js

```diff
@@ -0,0 +1,89 @@
+const controller = require('./get.controller')
+const { getSentencePlanComments } = require('../../common/data/sentencePlanningApi')
+
+jest.mock('../../common/data/sentencePlanningApi')
+
+const commentsEmpty = {}
+const commentsPresent = require('../../mockServer/sentencePlanComments/1.json')
+
+describe('getComments', () => {
+  const req = {
+    path: '/this/is/my/path',
+    params: {
+      planId: 1,
+    },
+    session: {
+      'x-auth-token': '1234',
+    },
+    body: {},
+    errors: {},
+    errorSummary: {},
+    renderInfo: null,
```

Comment:
beforeEach auto re-assigns this to an empty object. We never test a null/undefined scenario; is that important?

Reply:
Could be - I'll add one in